kfs-core/src/main/java/edu/arizona/kfs/fp/document/validation/impl/PaymentMethodRule.java

```diff
@@ -0,0 +1,183 @@
+package edu.arizona.kfs.fp.document.validation.impl;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.document.validation.impl.KfsMaintenanceDocumentRuleBase;
+import org.kuali.rice.krad.bo.PersistableBusinessObject;
+import org.kuali.rice.kns.document.MaintenanceDocument;
+import org.kuali.rice.krad.util.GlobalVariables;
+
+import edu.arizona.kfs.fp.businessobject.PaymentMethod;
+import edu.arizona.kfs.fp.businessobject.PaymentMethodChart;
+
+public class PaymentMethodRule extends KfsMaintenanceDocumentRuleBase {
```

Comment:
Please extend from org.kuali.rice.kns.maintenance.rules.MaintenanceDocumentRuleBase. KfsMaintenanceDocumentRuleBase is an empty class and is likely in KFS 6.0 just for backward compatibility.